go.mod

```diff
@@ -3,11 +3,12 @@ module github.com/ChainSafe/gossamer
 replace github.com/go-interpreter/wagon v0.0.0 => github.com/perlin-network/wagon v0.3.1-0.20180825141017-f8cb99b55a39
 
 require (
+	github.com/ChainSafe/chaindb v0.0.0-20200415142131-8c44b9255365
```

Comment:
can you update the version here ?

Reply:
I tried to change it to the tag (v0.1) but I get `go: errors parsing go.mod:
/home/elizabeth/go/src/github.com/ChainSafe/gossamer/go.mod:6: no matching versions for query "v0.1"` not sure how to fix it...

Reply:
done